visualizations/load-aware/store/parsers.py

```diff
@@ -233,11 +234,24 @@ def _parse_nodes_info(dirname, sutest_cluster=True):
 @ignore_file_not_found
 def _parse_ocp_version(dirname):
 
-    with open(register_important_file(dirname, pathlib.Path(artifact_dirnames.CLUSTER_CAPTURE_ENV_DIR) / "ocp_version.yml")) as f:
+    filename = artifact_paths.CLUSTER_CAPTURE_ENV_DIR / "ocp_version.yml"
+
+    with open(register_important_file(dirname, filename)) as f:
         sutest_ocp_version_yaml = yaml.safe_load(f)
 
     return sutest_ocp_version_yaml["openshiftVersion"]
 
+def _parse_file_locations(dirname):
+    file_locations = types.SimpleNamespace()
+
+    file_locations.trimaran_log = artifact_paths.LOAD_AWARE_SCALE_TEST_DIR / "trimaran_scheduler.log"
+    register_important_file(dirname, file_locations.trimaran_log)
```

Comment:
you need to add `trimaran_scheduler.log` in the list of important at the beginning of `parsers.py`
(`register_important_file` is a bit of a misnomer, it's more checking that the file is part of the list IIRC)